### PR TITLE
Add Ory

### PR DESCRIPTION
**Pull request by jonas-jonas**
Ory is a collection of Opensource microservices, solving authentication, user-management, authorization, and identity federation. All of these services are available for self-hosting or as a service and have strong, developer focused APIs & SDKs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,7 @@ The definition of developer-first for this repo is:
 * [Permit.io](https://www.permit.io/) - Permissions and access-control as a service.
 * [FusionAuth](https://fusionauth.io/) - Security-centric authentication as a Service, including self-hosting.
 * [userfront](https://userfront.com/) - Simplistic user authentication with pre-built back-office dashboard.
+* [Ory](https://ory.sh/) - Opensource identity & access solutions (also available as a service).
```
Comment (jonas-jonas) [inline README.md:78]:
```suggestion
* [Ory](https://ory.sh/) - Opensource identity & access-control solutions (also available as a service).
```

**Comment by agamm** [comment]
Closing until OP will review and make changes. @jonas-jonas

**Comment by jonas-jonas** [comment]
@agamm did you have any changes in mind? Not sure what else I could add?

**Comment by agamm** [comment]
@jonas-jonas sorry I didn't notice your changes before closing. Can you just insert it in the correct place alphabetically?

**Comment by jonas-jonas** [comment]
Sure thing! Done.

**Comment by agamm** [comment]
R < S, no? 😅 @jonas-jonas

**Comment by jonas-jonas** [comment]
> R < S, no? 😅 @jonas-jonas

LOL! yes of course :D

**Comment by agamm** [comment]
Thanks for the contribution!

**Pull request merged**